src/instructlab/model/chat.py

```diff
@@ -183,7 +189,6 @@ def chat(
     context,
     session,
     quick_question,
-    greedy_mode,
```

Comment:
The option for greedy_mode click option will need to be removed too

Reply:
Removed this on lines 141-144